src/Components/Boolist/Book.js

```diff
@@ -0,0 +1,42 @@
+import '../Style/Style.css';
+import NewBookForm from './NewBookForm';
+
+const Book = () => {
+  const data = [
+    {
+      id: 1,
+      title: 'Setup development environment',
+      author: ' Okpara Blessing',
+    },
+    {
+      id: 2,
+      title: 'Develop website and add content',
+      author: ' Okpara',
+    },
+    {
+      id: 3,
+      title: 'Deploy to live server',
+      author: ' Blessing',
+    },
+  ];
+  return (
+    <>
+
+      <div className="container">
+        {data.map((item) => (
+          <div className="eachBook" key={item.id}>
+            <h2>{item.title}</h2>
+            <p>
+              By
+              {item.author}
+            </p>
+            <button type="button" id="remove">Remove</button>
+          </div>
+        ))}
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure to create the `Book` component which will be the structure of how every single book will be rendered on the page. 

   - This `Book` component must take the book data as a prop and renders the `title` and the `author`.
   - It must also have a `button` for removing books but the button wouldn't remove the books yet, this is just a placeholder for now.

### The idea is that you need to create a new component for a single book and move all these parts of the code related to a single book structure to the newly created `Book` component.